src/porepy/numerics/fv/biot.py

```diff
@@ -311,14 +311,16 @@ def _discretize_mech(self, g, data):
             p = x[g.num_cells * gdim:]
 
         """
-        parameters = data[pp.PARAMETERS]
-        matrices = data[pp.DISCRETIZATION_MATRICES]
-        bound_mech = parameters["mechanics"]["bc"]
-        bound_flow = parameters["flow"]["bc"]
-        constit = parameters["mechanics"]["fourth_order_tensor"]
+        parameters_m = data[pp.PARAMETERS][self.mechanics_keyword]
+        parameters_f = data[pp.PARAMETERS][self.flow_keyword]
+        matrices_m = data[pp.DISCRETIZATION_MATRICES][self.mechanics_keyword]
+        matrices_f = data[pp.DISCRETIZATION_MATRICES][self.flow_keyword]
+        bound_mech = parameters_m["bc"]
+        bound_flow = parameters_f["bc"]
+        constit = parameters_m["fourth_order_tensor"]
 
-        eta = data.get("eta", 0)
-        inverter = data.get("inverter", None)
+        eta = parameters_m.get("mpsa_eta", fvutils.determine_eta(g))
+        inverter = parameters_m.get("inverter", None)
```

Comment:
Is it better to use 'mpsa_inverter'  to make the usage more specific? I added 'mpfa_inverter' to mpfa.py over in #188 , somehow that functionality had died during some updates.

Reply:
I think it then would be better with block_inverter or local_inverter